atomic_reactor/core.py

```diff
@@ -557,3 +557,11 @@ def image_exists(self, image_id):
             response = response is not None
         logger.debug("image exists: %s", response)
         return response
+
+    def get_info(self):
+        """
+        get info about used docker environment
+
+        :return: tuple of (`docker info`, `docker version`)
```

Comment:
True. Will rewrite. (this is actually really good suggestion: I had something in mind before I started to code, then changed it and forgot to update this)